src/TraceEvent/Stacks/CallTree.cs

```diff
@@ -431,8 +436,12 @@ private void AccumulateSumByID(CallTreeNode treeNode, Dictionary<int, CallTreeNo
         {
             if (recursionGuard.RequiresNewThread)
             {
+                var capturedThis = this;
+                var capturedTreeNode = treeNode;
+                var capturedCallersOnStack = callersOnStack;
+                var capturedRecursionGuard = recursionGuard;
```

Comment:
@sharwell, for each of these blocks, can you please add a comment as to why you're doing this?  It was not obvious to me until I read your comment above.

Reply:
Fixed